src/lib/components/admin/Settings/Models/ManageModelsModal.svelte

```
@@ -0,0 +1,110 @@
+<script>
+	import { toast } from 'svelte-sonner';
```

ESLint

[error] 'toast' is defined but never used.

(@typescript-eslint/no-unused-vars)